src/api/TokenIO.h

```diff
@@ -170,6 +170,39 @@ globalRpcErrorCallback:(OnError)globalRpcErrorCallback;
                           onSuccess:(OnSuccess)onSuccess
                             onError:(OnError)onError;
 
+#pragma mark - Recovery
+
+/**
+ * Begins recovery process for an alias. The verification message will be sent if the alias is valid.
```

Comment:
"Begins recovery process for an alias." sounds like user is just trying to re-demonstrate that they own an alias. But I think they're trying to show that they own a Token member by demonstrating that they own an alias. I think this sentence should change.

Maybe: "Begin member account recovery process by contacting alias." ?